### PR TITLE
PAE-344 - Adding support to enable the new course seat name

### PR DESCRIPTION
**Pull request by Jacatove**
**Description:**
This PR adds the functionality to change the course seat name in in get_course_seat_name by disabling/enabling the variable ENABLE_CUSTOM_COURSE_SEAT_NAME in settings. There is also`CUSTOM_SEAT_NAME` which allows to set the custom name. Let me show the following example:

`Seat in {CUSTOM_SEAT_NAME} <<Course Title>> course`
If `CUSTOM_SEAT_NAME = 'the Pearson Advanced`, then the resulting course seat name will be:
`Seat in the Pearson Advanced <<Course Title>> course`

### CODE REVIEW

File: ecommerce/courses/models.py [hunk 1]
```diff
@@ -129,6 +129,10 @@ def enrollment_code_product(self):
 
     def get_course_seat_name(self, certificate_type, id_verification_required):
         """ Returns the name for a course seat. """
+        if getattr(settings, 'ENABLE_NEW_COURSE_SEAT_NAME', False):
+            name = u'Seat in the Pearson Advance {} course'.format(self.name)
```
Comment (ivanvgh) [inline ecommerce/courses/models.py]: @Jacatove, what do you think, does this sound better?
```suggestion
            name = u'Seat in Pearson Advance {} course'.format(self.name)
```

Comment (Jacatove) [reply ecommerce/courses/models.py]: @ivanvgh I thought the same as you, but on the ticket is Seat in the Pearson...

**Review by ivanvgh — DISMISSED** [review]

File: ecommerce/courses/models.py [hunk 2]
```diff
@@ -129,6 +129,10 @@ def enrollment_code_product(self):
 
     def get_course_seat_name(self, certificate_type, id_verification_required):
         """ Returns the name for a course seat. """
+        if getattr(settings, 'ENABLE_NEW_COURSE_SEAT_NAME', False):
```
Comment (diegomillan) [inline ecommerce/courses/models.py]: We should use a setting name to suggest this is a custom change.
```suggestion
        if getattr(settings, 'ENABLE_CUSTOM_COURSE_SEAT_NAME', False):
```

**Review by diegomillan — CHANGES_REQUESTED** [review]

**Review by diegomillan — DISMISSED** [review]

**Review by ivanvgh — DISMISSED** [review]

**Review by diegomillan — DISMISSED** [review]

**Review by ivanvgh — DISMISSED** [review]

File: ecommerce/courses/models.py [hunk 3]
```diff
@@ -129,6 +129,14 @@ def enrollment_code_product(self):
 
     def get_course_seat_name(self, certificate_type, id_verification_required):
         """ Returns the name for a course seat. """
+        if (getattr(settings, 'ENABLE_CUSTOM_COURSE_SEAT_NAME', False) and
```
Comment (Squirrel18) [inline ecommerce/courses/models.py]: I think this setting is not necessary, just CUSTOM_SEAT_NAME is enough. @Jacatove

File: ecommerce/courses/models.py [hunk 4]
```diff
@@ -129,6 +129,14 @@ def enrollment_code_product(self):
...
+                getattr(settings, 'CUSTOM_SEAT_NAME', '')):
+            name = u'Seat in {} {} course'.format(
```
Comment (Squirrel18) [inline ecommerce/courses/models.py]: You don't need the Unicode character. @Jacatove

File: ecommerce/courses/models.py [hunk 5]
```diff
@@ -129,6 +129,14 @@ def enrollment_code_product(self):
...
+                getattr(settings, 'CUSTOM_SEAT_NAME'),
+                self.name
```
Comment (Squirrel18) [inline ecommerce/courses/models.py]: Trailing comma. @Jacatove

File: ecommerce/courses/models.py [hunk 6]
```diff
@@ -129,6 +129,14 @@ def enrollment_code_product(self):
...
+            )
+            return name
```
Comment (Squirrel18) [inline ecommerce/courses/models.py:138]: Add a blank line before this line. @Jacatove

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: ecommerce/courses/models.py (hunk 3 above)
Comment (Jacatove) [reply ecommerce/courses/models.py]: @Squirrel18 Yes one setting does the job. But I will rename it as `CUSTOM_COURSE_SEAT_NAME` to be a bit clearer.

File: ecommerce/courses/models.py (hunk 4 above)
Comment (Jacatove) [reply ecommerce/courses/models.py]: I used it to keep with the consistency within the method, the other formatted strings use it. @Squirrel18

**Comment by Jacatove** [comment]
@ivanvgh  @diegomillan @Squirrel18 I close this PR because I opened it against master. I add the link to the  [New PR](https://github.com/Pearson-Advance/ecommerce/pull/17) against Stage.

**Pull request closed without merging**